.idea/libraries/third_party.xml

```diff
@@ -0,0 +1,12 @@
+<component name="libraryTable">
```

Comment:
the .idea, bin, and *.iml files should not be in the repo.  The testing branch alraedy has a .gitignore file to exclude these.  You should remove them from your branch before merging.